.gitignore

```diff
@@ -0,0 +1,3 @@
+__pycache__/
+.vscode/
+mystbin/config.toml
```

Comment:
Can the config be put in the root directory?

Reply:
Resolved with 11c2528.